engine/index.js

```diff
@@ -421,13 +421,55 @@ handlebars.registerHelper('if_eq', function comparison(left, right, opts) { // N
   return opts.inverse(this);
 });
 
+// status partial
+const featureStatusContents = fs.readFileSync('src/tpl/featureStatusPartial.html', {
+  encoding: 'utf-8',
+});
+// links partial is needed as the feature page is build in slightly different
+// way (no summary), we might also switch off links when embedded
+const featureLinksContents = fs.readFileSync('src/tpl/featureLinksPartial.html', {
+  encoding: 'utf-8',
+});
+
 function buildIndex(status) {
   const templateContents = fs.readFileSync('src/tpl/index.html', {
     encoding: 'utf-8',
   });
+  handlebars.registerHelper('featureStatusName', function featureStatusName() {
```

Comment:
Ah, I didn't notice the function was bound by `handlebars`. Forget my comment then, let's just make the functions anonymous.